unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/SphereShape/ECSSphereShapeHandler.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using DCL.Controllers;
+using DCL.ECSRuntime;
+using DCL.Helpers;
+using DCL.Models;
+using UnityEngine;
+
+
+public class ECSSphereShapeComponentHandler : IECSComponentHandler<ECSShpereShape>
```

Comment:
could we have a "util" function for all those things that are common among all shape components?